### ISSUE TITLE
DateTimePicker Typing Year

### ISSUE THREAD

**Issue by PaitoAnderson** [telerik/kendo-react#251]
## I'm submitting a...
* Bug report

## Current behavior
If I click the year component in a `<DateTimePicker />` component and type "2019", it changes the year to 1920 and modifies and time.

## Expected behavior
I would expect to be able type the year.

## Minimal reproduction of the problem with instructions
https://www.telerik.com/kendo-react-ui/components/dateinputs/datetimepicker/controlled-state/#toc-controlling-the-value

Just click "2019" and then type "2019".

Before:
<img width="239" alt="Screen Shot 2019-05-16 at 11 45 50 AM" src="https://user-images.githubusercontent.com/4108718/57868575-3d30f680-77d1-11e9-8f0f-ecbc0dcadb9d.png">

After:
<img width="245" alt="Screen Shot 2019-05-16 at 11 54 04 AM" src="https://user-images.githubusercontent.com/4108718/57868645-5cc81f00-77d1-11e9-9b4c-c7e60ed870d3.png">

## What is the motivation or use case for changing the behavior?
Our users like to be able to type the date and time in a fast paced environment.

## Environment

Package versions:
```
├── @progress/kendo-data-query@1.5.1
├── @progress/kendo-drawing@1.5.12
├── @progress/kendo-react-buttons@2.11.0
├── @progress/kendo-react-dateinputs@2.11.0
├── @progress/kendo-react-dialogs@2.11.0
├── @progress/kendo-react-dropdowns@2.11.0
├── @progress/kendo-react-editor@2.11.0
├── @progress/kendo-react-grid@2.11.0
├── @progress/kendo-react-inputs@2.11.0
├── @progress/kendo-react-intl@2.11.0
├── @progress/kendo-react-layout@2.11.0
├── @progress/kendo-react-pdf@2.11.0
├── @progress/kendo-react-upload@2.11.0
├── @progress/kendo-theme-bootstrap@3.4.1
```

Browser:

- Chrome (desktop) version 74.0.3729.157

System:
- TypeScript version: 3.2.2
- Node version: v12.2.0 & v6.9.0
- Platform:  macOS

**Comment by Xizario** [comment]
I confirm this as a bug. Setting `format="g"` for the DatePicker, should workaround it until we fix it.

**Comment by Xizario** [comment]
Fixed in latest dev.
https://www.telerik.com/kendo-react-ui-develop/components/dateinputs/datetimepicker/controlled-state/#toc-controlling-the-value

**Comment by PaitoAnderson** [comment]
@Xizario Beautiful! Thanks!

**Comment by PaitoAnderson** [comment]
Fixed in 3.0.0.
